pkg/asset/installconfig/azure/validation.go

```diff
@@ -75,7 +75,16 @@ func ValidateInstanceType(client API, fieldPath *field.Path, region, instanceTyp
 			}
 			if cpus < float64(req.minimumVCpus) {
 				errMsg := fmt.Sprintf("instance type does not meet minimum resource requirements of %d vCPUs", req.minimumVCpus)
-				allErrs = append(allErrs, field.Invalid(fieldPath.Child("type"), instanceType, errMsg))
+				allErrs = append(allErrs, field.Invalid(fieldPath.Child("type", "vCPUs"), instanceType, errMsg))
```

Comment:
The field path should not be changed. This should point to a field in the install-config.yaml.